experimental/split_grpc/certs/local.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Could you add a README in this folder to explain why we have these secrets here, and commands to re-generate them if necessary?